service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -743,4 +743,8 @@ public DynamicList buildJudicialMessageDynamicList() {
     @JsonUnwrapped
     @Builder.Default
     private final SyntheticCaseSummary syntheticCaseSummary = SyntheticCaseSummary.builder().build();
+
+    public boolean hasSelectedTemporaryJudge(JudgeAndLegalAdvisor judge) {
```

Comment:
i just thought because this method is used in two different controllers - should i just place the logic back into if in each controller or what you think?